american-journal-of-medical-genetics.csl

```diff
@@ -24,13 +24,13 @@
   </info>
   <macro name="editor">
     <names variable="editor" delimiter=", ">
-      <name name-as-sort-order="all" sort-separator=", " initialize-with="" delimiter=", " delimiter-precedes-last="always"/>
+      <name name-as-sort-order="all" sort-separator=", " initialize-with="" delimiter=" " delimiter-precedes-last="always"/>
```

Comment:
Yes that's the correct documentation link/instructions for authors, you have to go to the "References" section to see the mention of brackets instead of parentheses. 

The reason this was brought to my attention is because we have a paper currently in review and the editor requested we fix our reference and in-text citation styles to match with the Instructions to Authors and said "please note Wiley has recently changed these."  So I'm assuming that recent journal articles are lagging behind and still using the old style since we've been given explicit instructions to change the styling from what is in place now.